root/edit/details/AddStandaloneRecording.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import ExpandedArtistCredit from '../../components/ExpandedArtistCredit';
+import formatTrackLength from
+  '../../static/scripts/common/utility/formatTrackLength';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AddStandaloneRecordingProps = {
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string | null,
+    +length: number | null,
+    +name: string,
+    +recording: RecordingT,
+    +video: boolean,
+  },
+};
+
+const AddStandaloneRecording =
+  ({edit}: {edit: AddStandaloneRecordingProps}) => {
+    const display = edit.display_data;
+    console.log(display);
+    return (
+      <>
+        <table className="details">
+          <tbody>
+            <tr>
+              <th>{addColonText(l('Recording'))}</th>
+              <td><EntityLink allowNew entity={display.recording} /></td>
+            </tr>
+          </tbody>
+        </table>
+        <table className="details add-standalone-recording">
+          <tbody>
+            <tr>
+              <th>{addColonText(l('Name'))}</th>
+              <td>
+                <EntityLink
+                  allowNew
+                  content={display.name}
+                  entity={display.recording}
+                  showDisambiguation
```

Comment:
Looks like we don't show the disambiguation for the name in production, just for "Recording" above. I think that makes more sense, since the disambiguation isn't part of the name.